### ISSUE TITLE
TypeError: Cannot read property 'default' of undefined

### ISSUE THREAD

**Issue by PhmNgocNghia** [vue-styleguidist/vue-styleguidist#196]
Reproduce repository : https://github.com/PhmNgocNghia/reproduce-styleguidist-bug
Reproduce instruction:
- Clone
- Install dependency
- npm run styleguide
- Error
How to fix:
Replace import state of child component to require('path').default

Not sure is it a bug with stlyeguidist or my project got misconfig webpack... It's seem like webpack doesn't transpile import statement with babel. idk

**Comment by elevatebart** [comment]
You bug is linked to the use by one of your dependencies of the beta of babel 7.
A fix is ready in this PR vue-styleguidist/vue-docgen-api#52
As soon as @rafaesc merges and releases you can update vue-docgen-api.

**Comment by elevatebart** [comment]
We might release the core and the cli-plugin later today.

**Comment by PhmNgocNghia** [comment]
Any temporary fix ?

**Comment by PhmNgocNghia** [comment]
I saw docgen api has been updated. Any temporary fix for core and cli plugin ?

**Comment by rafaesc** [comment]
@PhmNgocNghia  I have already updated vue-docgen-api

Try to install the last version of `vue-docgen-api`

**Comment by DanielPe05** [comment]
Is there a plan to update the versions on core / the cli plugin? I have this issue after a fresh install of the cli plugin.

**Comment by elevatebart** [comment]
released v0.1.4 of the plugin

**Comment by PhmNgocNghia** [comment]
```js
/home/phmngocnghia/Workspace/Cloudjet-KPI/cloudjet/cloudjet_ui/node_modules/vue-styleguidist/bin/styleguidist.js:48
let config;
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:442:10)
    at startup (node.js:136:18)
    at node.js:966:3
```

I got new error

**Comment by elevatebart** [comment]
what version of node are you running ?

**Comment by PhmNgocNghia** [comment]
wow it's v4.2.6. should have update it. gonna let u know if error is fixed or not

**Comment by elevatebart** [comment]
It should be fixed since node v6 I think.

**Comment by elevatebart** [comment]
Is it still testable in the same repo ?

**Comment by elevatebart** [comment]
Might be time do another issue, the two seem very unrelated ;)

**Comment by DanielPe05** [comment]
I can confirm that right after the update I see `Unknown language: "vue"` as well.

**Comment by cj** [comment]
@elevatebart I'm getting this error with the latest version: `"vue-docgen-api": "2.6.11"` and `"vue-styleguidist": "2.1.2"`

```
You can now view your style guide in the browser:

  Local:            http://localhost:6060/
  On your network:  http://192.168.1.134:6060/

Warning: Cannot parse src/components/AppButton.vue: TypeError: Cannot read property 'default' of undefined

It usually means that vue-docgen-api does not understand your source code or when using third-party libraries, try to file an issue here:
https://github.com/vue-styleguidist/vue-docgen-api/issues
```

This is with the basic example installed with https://github.com/vue-styleguidist/vue-cli-plugin-styleguidist.

**Comment by cj** [comment]
Looks like this is the cause https://github.com/vue-styleguidist/vue-docgen-api/pull/52/files#r245460928

**Comment by elevatebart** [comment]
Just fixed it with @cj temporary fix
available in vsg build 2.1.3.

Probably a better fix next week.
